unix_commands.sh

```diff
@@ -0,0 +1,14 @@
+echo Shaked >> a-1
+touch 1-b
+echo Shaked >> 1-b
+mkdir new
+cp 1-a new/
+mv 1-a 1-a-new
+gedit 1-a-new
+Shaked, cats
+echo Shaked, cats >> new/newFile
+cd new
+ls
+cd .. 
+cd -
+rm -r
```

Comment:
fix the command to be rm -rf new will recursively delete the directory new and all of its contents. The -r flag tells rm to recursively delete directories, and the -f flag tells rm to force the deletion, even if the directory is not empty.